libbeat/processors/add_cloud_metadata/provider_google_gce.go

```diff
@@ -63,11 +77,36 @@ var gceMetadataFetcher = provider{
 						"type": c.Str("machineType"),
 					},
 					"availability_zone": c.Str("zone"),
+					"orchestrator": s.Object{
+						"cluster": c.Dict(
+							"attributes",
+							s.Schema{
+								"name":       c.Str("cluster-name"),
```

Comment:
In case some instance that is not in a kubernetes cluster also has a `cluster-name` attribute,  wdyt about adding the `cluster-name` by now only if there is a kubeconfig?

Reply:
👍🏼